airbyte-integrations/connectors/source-wordpress/manifest.yaml

```diff
@@ -0,0 +1,4772 @@
+        schema:
+          $ref: "#/schemas/categories"
+    taxonomies:
+      type: DeclarativeStream
+      name: taxonomies
+      retriever:
+        type: SimpleRetriever
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            field_name: page
+            inject_into: request_parameter
+          pagination_strategy:
+            type: PageIncrement
+            page_size: 100
+            start_from_page: 1
+            inject_on_first_request: true
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: wp/v2/taxonomies
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+      primary_key:
+        - category
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/taxonomies"
+  base_requester:
+    type: HttpRequester
+    url_base: https://{{ config['domain'] }}/wp-json/
+    authenticator:
+      type: BasicHttpAuthenticator
+      password: "{{ config[\"password\"] }}"
+      username: "{{ config[\"username\"] }}"
+
+streams:
+  - $ref: "#/definitions/streams/users"
+  - $ref: "#/definitions/streams/posts"
+  - $ref: "#/definitions/streams/categories"
+  - $ref: "#/definitions/streams/plugins"
+  - $ref: "#/definitions/streams/blocks"
+  - $ref: "#/definitions/streams/comments"
+  - $ref: "#/definitions/streams/pages"
+  - $ref: "#/definitions/streams/tags"
+  - $ref: "#/definitions/streams/revisions"
+  - $ref: "#/definitions/streams/media"
+  - $ref: "#/definitions/streams/taxonomies"
+  - $ref: "#/definitions/streams/types"
+  - $ref: "#/definitions/streams/themes"
+  - $ref: "#/definitions/streams/statuses"
+  - $ref: "#/definitions/streams/search"
+  - $ref: "#/definitions/streams/settings"
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - domain
+      - password
+      - username
+      - parent_id
+    properties:
+      domain:
+        type: string
+        description: The domain of the WordPress site
+        name: domain
+        order: 0
+        title: Domain
+        default: http://biplab.great-site.net
+      password:
+        type: string
+        description: >-
+          Placeholder for basic HTTP auth password - should be set to empty
+          string
+        name: password
+        order: 1
+        title: Placeholder Password
+        default: x
+        airbyte_secret: true
+      username:
+        type: string
+        description: >-
+          Placeholder for basic HTTP auth username - should be set to empty
+          string
+        name: username
+        order: 2
+        title: Placeholder Username
+        default: x
+        airbyte_secret: true
+      parent_id:
+        type: string
+        order: 3
+        title: Parent Id
```

Comment:
What is Parent Id?

Reply:
parent id refers to the id of the original page for which the revisions are being fetched.